lib/BackbeatConsumer.js

```diff
@@ -196,10 +195,16 @@ class BackbeatConsumer extends EventEmitter {
     * is paused until the current queue of tasks are processed. Once the task
     * queue is empty, the current offset is committed and the fetch is resumed
     * to get the next batch of messages
+    * @param {Boolean} [disableConsumer] - optional field, if true, kafka
+    *   consumer should not subscribe to its topic
     * @return {this} current instance
     */
-    subscribe() {
-        this._consumer.subscribe([this._topic]);
+    subscribe(disableConsumer) {
+        if (!disableConsumer) {
+            this._consumer.subscribe([this._topic]);
+        } else {
+            this._log.info(`consumer not subcribed to topic ${this._topic}`);
```

Comment:
At this level I would rather log in debug mode, as the message is likely obscure for a normal user. Such message would better fit at the service level (stating that it's paused, for example).